apps/country-tracker/features/home/components/home-screen.tsx

```diff
@@ -1,59 +1,56 @@
-import { ThemedView } from "@/components/ThemedView";
+import { Box } from "@/components/ui/box";
+import { Heading } from "@/components/ui/heading";
+import { Input, InputField, InputIcon, InputSlot } from "@/components/ui/input";
 import { Text } from "@/components/ui/text";
-import i18n from "@/i18n";
+import { Search } from "lucide-react-native";
+import type { ListRenderItem } from "react-native";
+import { FlatList } from "react-native";
+
+type CountryItem = {
+  id: string;
+  country: string;
+  flag: string;
+  lastVisitDate: string;
+};
+
+const data: CountryItem[] = [
+  { id: "1", country: "France", flag: "🇫🇷", lastVisitDate: "2023-01-15" },
+  { id: "2", country: "Japan", flag: "🇯🇵", lastVisitDate: "2022-12-05" },
+];
 
 export default function HomeScreen() {
+  const renderItem: ListRenderItem<CountryItem> = ({ item }) => (
+    <Box className="mt-4 flex flex-row justify-between border-gray-100 border-b p-2">
+      <Box className="flex flex-1 flex-row gap-2">
+        <Text>{item.flag}</Text>
+        <Text>{item.country}</Text>
+      </Box>
+      <Text className="font-mono text-gray-500">{item.lastVisitDate}</Text>
+    </Box>
+  );
+
   return (
-    <ParallaxScrollView
-      headerBackgroundColor={{ light: "#A1CEDC", dark: "#1D3D47" }}
-      headerImage={
-        <Image
-          source={require("@/assets/images/partial-react-logo.png")}
-          className="absolute bottom-0 left-0 h-[178px] w-[290px]"
-        />
-      }
-    >
-      <ThemedView className="flex-row items-center gap-2">
-        <Text bold size="4xl">
-          {i18n.t("welcome")}!
-        </Text>
-        <HelloWave />
-      </ThemedView>
-      <ThemedView className="mb-2 gap-2">
-        <Text bold size="2xl">
-          Step 1: Try it
-        </Text>
-        <Text>
-          Edit <Text bold>app/(tabs)/index.tsx</Text> to see changes. Press{" "}
-          <Text bold>
-            {Platform.select({ ios: "cmd + d", android: "cmd + m" })}
-          </Text>{" "}
-          to open developer tools.
-        </Text>
-      </ThemedView>
-      <ThemedView className="mb-2 gap-2">
-        <Text bold size="2xl">
-          Step 2: Explore
-        </Text>
-        <Text>
-          Tap the Explore tab to learn more about what's included in this
-          starter app.
-        </Text>
-      </ThemedView>
-      <ThemedView className="mb-2 gap-2">
-        <Text bold size="2xl">
-          Step 3: Get a fresh start
-        </Text>
-        <Text>
-          When you're ready, run <Text bold>npm run reset-project</Text> to get
-          a fresh <Text bold>app</Text> directory. This will move the current{" "}
-          <Text bold>app</Text> to <Text bold>app-example</Text>.
-        </Text>
-      </ThemedView>
+    <ParallaxScrollView>
+      <FlatList
+        data={data}
+        keyExtractor={(item) => item.id}
+        renderItem={renderItem}
+        ListHeaderComponent={
+          <Box className="flex gap-4">
```

Comment:
> gap-4는 되는데 gap-2는 안되는 경우 해결방안이 있을지 궁금합니다~

요 이슈 발생 위치가 어딘가요? 1️⃣
L40 gap-4 👉 gap-2 적용은 확인했습니다!

Reply:
앗 이런... 국기 이모지랑 국가명 사이 gap 적용이 안되는군요

---

> L40 gap-4 👉 gap-2 적용은 확인했습니다!

요건 `pnpm run clean`하기 전 web에서만 확인했습니다
이 상태에서 iOS 구동시키니
```
Invariant Violation: Tried to register two views with the same name RNSVGSvgView, js engine: hermes 
    at ContextNavigator (http://172.30.1.98:8081/index.ts.bundle//&platform=ios&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.baseUrl=%2Fapp-factory%2Fcountry-tracker&transform.routerRoot=app:30319:24)
```

요런게 뜨길래 (제 앱 svg 관련 캐싱으로 보여서요)
`pnpm run clean` 이후 다시 확인해보니
국기 이모지랑 국가명 사이 gap 적용이 안되는군요

---

gap 적용 안됨 확인 이후
제 로컬 터미널에 두가지가 뜹니다
둘 다 글루스택의 코드 스니펫으로 보이는데요...🥲

1. cycle 지우시오
```
 WARN  Require cycle: ../../node_modules/@internationalized/date/dist/CalendarDate.main.js -> ../../node_modules/@internationalized/date/dist/manipulation.main.js -> ../../node_modules/@internationalized/date/dist/conversion.main.js -> ../../node_modules/@internationalized/date/dist/CalendarDate.main.js

Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle. 
  factory (../../node_modules/@internationalized/date/dist/conversion.main.js:1:33)
  factory (../../node_modules/@internationalized/date/dist/manipulation.main.js:1:33)
λ  WARN  Require cycle: ../../node_modules/@internationalized/date/dist/manipulation.main.js -> ../../node_modules/@internationalized/date/dist/conversion.main.js -> ../../node_modules/@internationalized/date/dist/manipulation.main.js

Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle. 
  factory (../../node_modules/@internationalized/date/dist/conversion.main.js:2:33)
  factory (../../node_modules/@internationalized/date/dist/manipulation.main.js:1:33)
λ  WARN  Require cycle: ../../node_modules/@internationalized/date/dist/CalendarDate.main.js -> ../../node_modules/@internationalized/date/dist/manipulation.main.js -> ../../node_modules/@internationalized/date/dist/conversion.main.js -> ../../node_modules/@internationalized/date/dist/GregorianCalendar.main.js -> ../../node_modules/@internationalized/date/dist/CalendarDate.main.js

Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle. 
  factory (../../node_modules/@internationalized/date/dist/GregorianCalendar.main.js:1:33)
  factory (../../node_modules/@internationalized/date/dist/conversion.main.js:3:33)
λ  WARN  Require cycle: ../../node_modules/@internationalized/date/dist/CalendarDate.main.js -> ../../node_modules/@internationalized/date/dist/manipulation.main.js -> ../../node_modules/@internationalized/date/dist/conversion.main.js -> ../../node_modules/@internationalized/date/dist/GregorianCalendar.main.js -> ../../node_modules/@internationalized/date/dist/utils.main.js -> ../../node_modules/@internationalized/date/dist/CalendarDate.main.js

Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle. 
  factory (../../node_modules/@internationalized/date/dist/utils.main.js:1:33)
  factory (../../node_modules/@internationalized/date/dist/GregorianCalendar.main.js:2:33)
λ  WARN  Require cycle: ../../node_modules/@internationalized/date/dist/conversion.main.js -> ../../node_modules/@internationalized/date/dist/queries.main.js -> ../../node_modules/@internationalized/date/dist/conversion.main.js

Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle. 
  factory (../../node_modules/@internationalized/date/dist/queries.main.js:1:33)
  factory (../../node_modules/@internationalized/date/dist/conversion.main.js:4:33)
λ  WARN  Require cycle: ../../node_modules/@internationalized/date/dist/CalendarDate.main.js -> ../../node_modules/@internationalized/date/dist/string.main.js -> ../../node_modules/@internationalized/date/dist/CalendarDate.main.js

Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle. 
  factory (../../node_modules/@internationalized/date/dist/string.main.js:1:33)
  factory (../../node_modules/@internationalized/date/dist/CalendarDate.main.js:3:33)
```

2. `VirtualizedLists` nested 하지마시오
```
VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality - use another VirtualizedList-backed container instead. 
    at VirtualizedList (http://172.30.1.98:8081/index.ts.bundle//&platform=ios&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.baseUrl=%2Fapp-factory%2Fcountry-tracker&transform.routerRoot=app:72449:36)
    at CssInterop.VirtualizedList (http://172.30.1.98:8081/index.ts.bundle//&platform=ios&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.baseUrl=%2Fapp-factory%2Fcountry-tracker&transform.routerRoot=app:10662:45)
    at FlatList (http://172.30.1.98:8081/index.ts.bundle//&platform=ios&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.baseUrl=%2Fapp-factory%2Fcountry-tracker&transform.routerRoot=app:71799:36)
    at CssInterop.FlatList (http://172.30.1.98:8081/index.ts.bundle//&platform=ios&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.baseUrl=%2Fapp-factory%2Fcountry-tracker&transform.routerRoot=app:10662:45)
    at RCTView
    at View (http://172.30.1.98:8081/index.ts.bundle//&platform=ios&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.baseUrl=%2Fapp-factory%2Fcountry-tracker&transform.routerRoot=app:62510:43)
    at CssInterop.View (http://172.30.1.98:8081/index.ts.bundle//&platform=ios&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.baseUrl=%2Fapp-factory%2Fcountry-tracker&transform.routerRoot=app:10662:45)
    at ThemedView (http://172.30.1.98:8081/index.ts.bundle//&platform=ios&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.baseUrl=%2Fapp-factory%2Fcountry-tracker&transform.routerRoot=app:284567:21)
    at RCTScrollContentView
    at RCTScrollView
    at ScrollView (http://172.30.1.98:8081/index.ts.bundle//&platform=ios&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.baseUrl=%2Fapp-factory%2Fcountry-tracker&transform.routerRoot=app:77354:36)
    at ScrollView
    at CssInterop.ScrollView (http://172.30.1.98:8081/index.ts.bundle//&platform=ios&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.baseUrl=%2Fapp-factory%2Fcountry-tracker&transform.routerRoot=app:10662:45)
```

---

마지막으로 은광님 앱 > `VirtualizedList` 관련
`cssInterop` ts 에러가 보이네요

```
/app-factory/apps/country-tracker/components/ui/select/index.tsx
/app-factory/apps/country-tracker/components/ui/select/select-actionsheet.tsx
```

<img width="1371" alt="image" src="https://github.com/user-attachments/assets/14e992e7-ec40-460c-b1e8-bbe6c5fea3b7">
<img width="1375" alt="image" src="https://github.com/user-attachments/assets/285f3ca7-aec4-4195-86dd-afe66c80d383">

Reply:
이것들은 사이클 디펜던시 글루스택에서 처리해줘야해서 문제 없을듯요~

VirtualizedLists nested 이건 고쳐봐야겠네요 FlatList 를 글루스택껄 쓰던지..